src/auth/oauth.ts

```diff
@@ -91,6 +91,19 @@ export interface ClientCredentialsGrantRequest extends ClientCredentials {
   audience: string;
 }
 
+interface AuthorizationDetails {
```

Comment:
Should this be exported?

Reply:
Yes, this should have been exported. Thanks for pointing it out, will update.